setup.py

```diff
@@ -65,6 +65,8 @@
         'contextvars ; python_version >= "3.6" and python_version < "3.7"',
         'importlib-metadata ; python_version < "3.8"',
         "psutil>=5,<6",
+        'urllib3 < 1.25 ; python_version < "3.5"',
+        'urllib3 < 2 ; python_version >= "3.5"',
```

Comment:
Is there a reason to limit the version of urllib3 with py3.5+?

Reply:
It was part of your requirements.txt to limit the versionof the urllib3[secure].
i'll just update with a new pull request as the deprecation doesn't have effect on urllib3 1.25.x or earlier

Reply:
Thanks for pointing that out. I need to investigate that.